tests/Unit/Ads/AccountServiceTest.php

```diff
@@ -171,8 +180,14 @@ public function test_get_connected_account_incomplete() {
 			->willReturn( self::TEST_ACCOUNT_ID );
 
 		$this->options->method( 'get' )
-			->with( OptionsInterface::ADS_ACCOUNT_CURRENCY )
-			->willReturn( self::TEST_CURRENCY );
+			->withConsecutive(
+				[ OptionsInterface::ADS_ACCOUNT_OCID ],
+				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ]
+			)
+			->willReturnOnConsecutiveCalls(
+				self::TEST_ACCOUNT_OCID,
+				self::TEST_CURRENCY
+			);
```

Comment:
@asvinb same again here

```suggestion
			->withConsecutive(
				[ OptionsInterface::ADS_ACCOUNT_OCID ],
				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ],
				[ OptionsInterface::ADS_ACCOUNT_CURRENCY ]
			)
			->willReturnOnConsecutiveCalls(
				self::TEST_ACCOUNT_OCID,
				self::TEST_CURRENCY,
				self::TEST_CURRENCY
			);
```